recipes/tinkerforge-bindings/all/conanfile.py

```diff
@@ -70,13 +70,13 @@ def package(self):
         cmake.install()
 
     def package_info(self):
-        self.cpp_info.set_property("cmake_target_name", "tinkerforge::bindings")
+        self.cpp_info.set_property("cmake_target_name", "tinkerforge-bindings")
```

Comment:
My intention was to have target equal to `tinkerforge::bindings` (proposing all Tinker forge libraries in `tinkerforge` namespace).